### PR TITLE
✨ Create Subheader Component

### PR DESCRIPTION
**Pull request by mrana17**
- SubHeader Component with Data
- Styled SubHeader

### CODE REVIEW

File: components/subHeader/subHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import styles from "./subHeader.module.css";
+
+export type SubHeaderProps = {
+  infoOne: string;
+  infoOneNumber: string;
```
Comment (mariegross) [inline components/subHeader/subHeader.tsx]: I'd suggest to use type number:
```suggestion
  infoOneNumber: number;
```

File: components/subHeader/subHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  infoTwo: string;
+  infoTwoNumber: string;
+  infoThree: string;
+  infoThreeNumber: string;
```
Comment (mariegross) [inline components/subHeader/subHeader.tsx]: If you convert the amounts into arrays, the use becomes more variable: You can use the numbers for calculations and maybe even use different currencies at some point. You have more control, too :)
```suggestion
  infoTwoNumber: [number, string];
  infoThree: string;
  infoThreeNumber: [number, string];
```
I'm no pro in TS, but I think you can do it like this :)
https://www.typescriptlang.org/docs/handbook/basic-types.html#tuple

**Review by mariegross — COMMENTED** [review]
Just added some small suggestions :)

**Comment by mrana17** [comment]
> Just added some small suggestions :)

Thank you for your suggestions! You are looking like a typescript pro to me!

**Pull request merged**